cmd_plus.go

```diff
@@ -52,14 +52,22 @@ func (c *CmdPlus) Kill() error {
 	return nil
 }
 
+// GetOutput returns the output thus far
+func (c *CmdPlus) GetOutput() string {
+	c.mutex.Lock()
+	result := c.output
+	c.mutex.Unlock()
```

Comment:
What about:

```go
c.mutex.Lock()
defer c.mutex.Unlock()
return c.output
```

Also, is this mutex used elsewhere? Might make sense to use our own mutex here. Not too sure since the times I did such things manually are decades in the past...